docs/DeveloperGuide.md

```diff
@@ -200,6 +198,44 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Loading pre-existing data
+The loading of pre-existing data is mainly handled by the `filemanager` and `model` components. The main method calls `loadFileData(instrumentManager)` 
+in the `Storage` class which implements `InstrumentDecoder#readFile(instrumentManager, fileData)`. This method calls 
+`addSavedInstrumentToList(instrumentManager, textSegment)` for each pre-existing instrument which will add the corresponding 
+instrument in the `InstrumentManager` through calling the `addXYZToList` method in the XYZDecoder classes. In the event
+the instrument is not one of the 4 types of instruments, the `InstrumentDecoder` will throw a new `InvalidInstrumentInFileError`
+and display the corresponding error message.
+
+The figures below represents the sequence diagrams when the user loads a pre-existing crypto:
+
+<img src="images/FileManagerSeqBetweenStorageAndDecoder.png" width="800"/>
```

Comment:
<img width="921" alt="ss1" src="https://user-images.githubusercontent.com/60163283/139537403-3e02efe6-549e-4e60-a2d1-be326ac1cdc8.PNG">

I think you have to create a ref box in there to reference the bottom image. The bottom image is also formatted weirdly. Then the bottom image is to be encompassed in a sd ref frame